### DISCUSSION TITLE
第4节课作业(11班)

### DISCUSSION THREAD

**Discussion by keesh0410** [InternLM/Tutorial#239] [作业]
🐱：基础作业参考答案：https://github.com/InternLM/tutorial/blob/main/xtuner/self.md

**Comment by seifer08ms** [comment]
https://zhuanlan.zhihu.com/p/677133280?

**Comment by LichKing-W** [comment]
https://zhuanlan.zhihu.com/p/677349064

**Comment by zhanghui-china** [comment]
基础作业
[https://zhuanlan.zhihu.com/p/677258451](url)
进阶作业
[https://zhuanlan.zhihu.com/p/677362346](url)

**Comment by qianlian-mozi** [comment]
https://github.com/qianlian-mozi/internlm_practising_20240106/tree/main/lesson_4_homework

**Comment by BarryYin** [comment]
https://github.com/BarryYin/internLM/blob/main/%E7%AC%AC%E5%9B%9B%E8%8A%82%E4%BD%9C%E4%B8%9A.md

**Comment by Kongsea** [comment]
基础作业：https://blog.csdn.net/deeepsea/article/details/135560122

**Comment by horacehht** [comment]
https://blog.csdn.net/m0_50123831/article/details/135555893

**Comment by Shengshenlan** [comment]
https://github.com/Shengshenlan/tutorial/blob/master/homework/4/%E7%AC%AC%E5%9B%9B%E8%8A%82%E8%AF%BE%E4%BD%9C%E4%B8%9A.md

**Comment by JustLike1** [comment]
https://blog.csdn.net/qq_45607888/article/details/135561948

**Comment by wuyingxue1230** [comment]
https://zhuanlan.zhihu.com/p/677337813?

**Comment by wwewwt** [comment]
https://github.com/wwewwt/-InternLM/wiki/4%E3%80%81%E4%BD%9C%E4%B8%9A

**Comment by morning-start** [comment]
https://flowus.cn/morningstar/share/07617a21-98ac-4f1d-9278-1e5225882493
【FlowUs 息流】XTuner InternLM-Chat 个人小助手认知微调实践

**Comment by lvoooo** [comment]
https://github.com/lvoooo/internLM-learning/blob/main/4.work.md

**Comment by Vacili-f** [comment]
https://github.com/Vacili-f/InternLM/blob/main/%E4%BD%9C%E4%B8%9A/%E7%AC%AC%E5%9B%9B%E8%8A%82%E8%AF%BE%E4%BD%9C%E4%B8%9A.md

**Comment by Lyusijie** [comment]
https://blog.csdn.net/weixin_44986362/article/details/135591932?spm=1001.2014.3001.5501

**Comment by Everfighting** [comment]
https://github.com/Everfighting/InternLM_learning/blob/main/chapter4/chapter_4.md

**Comment by codefd** [comment]
https://blog.csdn.net/googteo/article/details/135567928?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135567928%22%2C%22source%22%3A%22googteo%22%7D

**Comment by GMYelims** [comment]
https://blog.csdn.net/weixin_42112760/article/details/135536178?spm=1001.2014.3001.5502

**Comment by GMYelims** [comment]
https://blog.csdn.net/weixin_42112760/article/details/135536178?spm=1001.2014.3001.5502

**Comment by Tangent-90C** [comment]
https://github.com/Tangent-90C/public-note/tree/main/%E5%BE%AE%E8%B0%83%E4%B8%AA%E8%87%AA%E5%B7%B1%E7%9A%84%E7%A7%98%E4%B9%A6

**Comment by keesh0410** [comment]
https://zhuanlan.zhihu.com/p/679127117
